testing-guidance/workflow-tests/animation/testassets/example-asset-screenshots.md

```diff
@@ -0,0 +1,26 @@
+# Purpose
+This reference file is intended to show what example assets looked like when generated. The reason to do this is that systems can be updated causing example assets to go out of date. Having screenshots of their setup allows for rebuilding of assets in new formats when upgrading is not possible.
+
+# Files
+
+## TestSC.scriptcanvas
+Example Script Canvas generated using **thirdpersonmovement.inputbindings** from the **Starting Point Input Gem**. 
```

Comment:
Similarly to the earlier comment, changing *generated using* to for example *created referencing* would avoid potential confusion, since the Script Canvas graph was created manually and the thirdpersonmovement.inputbindings only had its Input Event referenced (move forward).

*Example Script Canvas **generated using** **thirdpersonmovement.inputbindings** from the **Starting Point Input Gem**.* > *Example Script Canvas **created referencing** **thirdpersonmovement.inputbindings** from the **Starting Point Input Gem**.*